CHANGELOG.md

```diff
@@ -17,7 +17,7 @@ The repository at this stage should be generally complete with respect to infras
 
 With respect to infrastructure, the repository contains the following components:
 
-- detailed documentation in the `docs` directory explaining how to interact with the OpenScPCA project and how to set up and run analyses (published at https://openscpca.readthedocs.io/)
+- detailed documentation in the `docs` directory explaining how to interact with the OpenScPCA project and how to set up and run analyses (published at https://openscpca.readthedocs.io/en/latest/)
```

Comment:
Since this url is in the text, I would leave it as the bare url (not even a trailing slash). 
```suggestion
- detailed documentation in the `docs` directory explaining how to interact with the OpenScPCA project and how to set up and run analyses (published at https://openscpca.readthedocs.io) 
```